docs/content_management/pages/create_custom_page_block.md

```diff
@@ -132,5 +132,8 @@ The following UDW configuration is used so only an `event` typed content item ca
 
 For more information, see [UDW configuration](browser.md#udw-configuration).
 
-Your custom page block is now ready. 
-Before you can use it in Page Builder, you must [enable it in Page field settings]([[= user_doc =]]/content_management/configure_ct_field_settings/#block-display).
+Your custom page block is now ready.
+
+!!! caution
+
+    Before you can use the block in Page Builder, it needs to be enabled. This can be done manually in [Page field settings]([[= user_doc =]]/content_management/configure_ct_field_settings/#block-display) or by using the migration action [`add_block_to_available_blocks`](data_migration_actions.md#content-types).
```

Comment:
```suggestion
   To use the new block in Page Builder, add it to the list of available blocks in a given content type's settings. This can be done manually in [Page field settings]([[= user_doc =]]/content_management/configure_ct_field_settings/#block-display) or by using the migration action [`add_block_to_available_blocks`](data_migration_actions.md#content-types).
```

I'd expand the meaning of "enabled" - it's not an on/off switch for the whole system, it's an allowlist of blocks available for every Content Type.

I'm not sure about my suggestion, but this is the direction I'd choose.